connect/models/param.py

```diff
@@ -48,6 +48,9 @@ class Param(BaseModel):
     value_choices = None  # type: Optional[List[ValueChoice]]
     """ (List[str]|None) Available dropdown choices for parameter. """
 
+    structured_value = None  # type: Optional[str]
```

Comment:
`Optional[str]` and in comment `List[str]|None`

Reply:
fixed, thanks!